### PR TITLE
Da page curvy lines

### PR DESCRIPTION
**Pull request by Demoivre02**
## 1. Objective
Explain `WHY` this PR is required e.g What feature it adds, what problem it solves, ...

Related Link(s)
- [TaskID](#)
- [Issue ID](#)

## 2. Description of change
A description of `WHAT` changed in the codebase. Additionally add the reasoning if it's complex or abstract. Ermm.. you don't want to pour the entire diff, that's why there is `git diff`

## 3. Quality assurance

Explain how to manually test this feature. 

## 4. Operations impact
List the steps that must be taken for this PR to work e.g `npm install`, `yarn`, `add API_URL to environment variables`, etc

## 5. Business impact

If there is any customer-facing changes such as new features or API/UI changes, list them here.

## 6. Priority of Change

Normal, High or Immediate

### CODE REVIEW

**Comment by the-jossie** [comment]
@Demoivre02 please point your MR to the `staging` branch instead of `master`

**Comment by Demoivre02** [comment]
> 

done

File: src/templates/home/header.jsx [hunk 1]
```diff
@@ -24,48 +24,88 @@ export const HomeHeader = () => {
...
...592C31.2132 79.2358 27.2063 79.3896 23.2689 79.6902C21.4867 79.8088 19.7142 79.8564 17.9284 79.8199C17.038 79.8107 16.1491 79.7719 15.2562 79.7536C14.3442 79.7015 13.534 79.6996 12.5098 79.6296C11.6044 79.566 10.6089 79.4588 9.48026 79.115C8.91469 78.9385 8.32015 78.692 7.73197 78.3266C7.14628 77.9703 6.5504 77.4704 6.07411 76.9085C4.98505 75.6073 4.5459 74.079 4.38872 72.9025C4.21909 71.6806 4.26032 70.6936 4.3042 69.8229C4.47726 66.8288 5.42203 63.9447 6.23823 61.1252C6.42605 60.4589 6.23633 59.5548 6.46613 58.8282C6.65076 58.2214 6.96107 57.219 7.31827 56.6721C7.47712 56.4334 7.65903 56.2078 7.66185 55.9338C7.70339 54.6637 8.53352 54.6309 10.6992 55.8023C11.18 56.0606 11.8701 56.1931 12.1298 56.5706C12.1439 56.5862 12.163 56.62 12.177 56.6357C13.3056 57.8674 12.4235 58.4218 12.2252 59.2276C11.2723 63.1482 10.6694 67.2068 10.6194 71.1819C10.5908 72.3216 10.8017 73.054 10.943 73.142C11.0248 73.2269 11.1223 73.2977 11.4031 73.3962C11.6813 73.4857 12.0776 73.5427 12.5661 73.5452C13.105 73.5533 13.7217 73.2767 14.4276 73.5024L14.416 73.4958C15.1479 73.4607 15.8831 73.3662 16.6002 73.3839C19.3442 73.4988 22.2031 73.2505 24.8422 73.2673C24.9313 73.2721 25.0682 73.1663 25.2208 73.1536Z"
```
Comment (the-jossie) [inline src/templates/home/header.jsx]: @Demoivre02 I believe you'd want to save this svg as an asset and import it here instead

File: src/templates/home/header.jsx [hunk 2]
```diff
@@ -24,48 +24,88 @@ export const HomeHeader = () => {
...
....8736 71.9222 41.3476 68.5635 36.566C67.0562 34.3647 65.9128 31.8564 65.3076 29.3708C65.0262 28.1342 64.8745 26.8957 64.9015 25.7358C64.9703 24.5978 65.2274 23.5582 65.6867 22.7439C66.1461 21.9295 66.7754 21.3384 67.7066 20.9301C68.6277 20.5115 69.5897 20.3083 70.5342 20.3945C72.4639 20.608 74.7561 21.9662 76.5928 23.897C78.4753 25.7822 80.0283 28.1217 81.2104 30.526L81.6406 31.4425L82.0763 32.4465L82.9155 34.4526C83.4605 35.7923 83.9516 37.1287 84.4106 38.4631C86.3755 44.3413 87.1596 50.2934 86.2235 55.9574L86.2401 55.8615C85.6088 58.8047 84.5606 61.6839 82.8596 64.2913C82.1406 65.6514 81.054 66.8441 80.1462 68.1153L78.518 69.8753C78.2413 70.1684 77.9855 70.4723 77.6986 70.755L75.084 73.1233L73.2176 74.5398L72.2784 75.2574L71.2681 75.9029L69.2582 77.1946C66.4441 78.7571 63.5372 80.2462 60.3968 81.3146C57.3145 82.4931 54.0512 83.2734 50.7752 83.9078C49.1221 84.1176 47.4823 84.4637 45.8051 84.5365L43.3115 84.7245L39.4875 84.7837L38.2061 84.7353L35.6536 84.6486C33.9479 84.4777 32.2467 84.4136 30.5564 84.1662L28.0185 83.8385L25.4971 83.415C23.8182 83.1586 22.1481 82.7576 20.4846 82.4247C13.8401 80.9388 7.39182 78.884 1.12787 76.6371C0.154927 76.3072 -1.12489 75.6975 -2.55989 75.1558Z"
```
Comment (the-jossie) [inline src/templates/home/header.jsx]: ditto

File: src/pages/resources.js [hunk 3]
```diff
@@ -3,11 +3,42 @@ import React from "react";
 import Layout from "components/layout";
 import { Newsletter } from "components";
 import Container from "components/container";
+import ArrowIcon from "assets/images/svgs/arrow.svg";
 import ctl from "@netlify/classnames-template-literals";
 
 import { ResourceCategory, ResourcesHeader } from "templates/resources";
 import { graphql, useStaticQuery } from "gatsby";
 
+const arrowsContainerStyle = ctl(`
```
Comment (the-jossie) [inline src/pages/resources.js:12]: I'd suggest we move the styles below the component itself, should in case the styles get long in the future.

**Review by the-jossie — CHANGES_REQUESTED** [review]

**Pull request merged**